packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -93,6 +93,13 @@ export interface Ec2EnvironmentProps {
    * @default - CONNECT_SSH
    */
   readonly connectionType?: ConnectionType
+
+  /**
+   * The AMI alias or SSM path of the image to use
+   *
+   * @default - Cloud9 default AMI (currently amazonlinux-1-x86_64)
+   */
+  readonly imageId?: string;
```

Comment:
Since there are only a couple valid options I think we should use an enum here.

https://github.com/aws/aws-cdk/blob/v1-main/docs/DESIGN_GUIDELINES.md#enums

From the CFN docs it looks like you can specify an AMI alias or a _valid_ SSM path, but it doesn't looks like there is any real difference? If there isn't I would just default to using the alias and not giving the option for the SSM path. We could also use the second example in the linked docs so a user could provide it manually if they really wanted

```ts
export class MyOption {
  public static COMMON_OPTION_1 = new MyOption('common.option-1');
  public static COMMON_OPTION_2 = new MyOption('common.option-2');

  public static custom(value: string) {
    return new MyOption(value);
  }

  // 'protected' iso. 'private' so that someone that really wants to can still
  // do subclassing. But maybe might as well be private.
  protected constructor(public readonly value: string) { }
}

// Usage
new BoomBoom(this, 'Boom', {
  option: MyOption.COMMON_OPTION_1
});

new BoomBoom(this, 'Boom', {
  option: MyOption.custom('my-value')
});
```